test/minikube/minikube_upgrade_helm_test.go

```diff
@@ -138,8 +139,12 @@ func upgradeDatabaseTest(t *testing.T, fromHelmVersion string, updateOptions *Up
 	// all jobs need to be deleted before an upgrade can be performed
 	// so far we have not found an automated way to delete them as part of a pre-upgrade hook
 	// if we find it, this line can be removed and the test should still pass
-	testlib.DeletePod(t, namespaceName, "jobs/job-lb-policy-nearest")
```

Comment:
I think removing this reliance on knowing all the jobs will make the tests more robust going forward.